### PR TITLE
Generating CI config for Knative Eventing release-v0.18.4

### PR DESCRIPTION
**Pull request by cardil**
As title says.

### CODE REVIEW

File: ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.18.4__46.yaml [hunk 1]
```diff
@@ -249,30 +257,14 @@ tag_specification:
 test_binary_build_commands: make test-install
 tests:
 - as: e2e-aws-ocp-46
-  steps:
+  commands: make test-e2e
+  openshift_installer_src:
```
Comment (matzew) [inline ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.18.4__46.yaml:261]: oh, these are the OLD template

**Comment by cardil** [comment]
This is required for https://github.com/openshift/release/pull/13751 to be able to sync

**Comment by matzew** [comment]
/hold

use the modified script from this PR @cardil 
https://github.com/openshift/knative-eventing/pull/988

**Comment by cardil** [comment]
Now, it looks much better

/unhold

**Comment by cardil** [comment]
It's required for openshift-knative/serverless-operator#697

**Comment by cardil** [comment]
Test was failing due to openshift/knative-eventing#985

Now, it's merged we can do a:

/retest

**Pull request merged**